ktor-server/ktor-server-core/jvmAndNix/src/io/ktor/server/application/plugins/api/CallFailed.kt

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2014-2021 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.server.application.plugins.api
+
+import io.ktor.server.application.*
+
+public object CallFailed : Event<suspend (call: ApplicationCall, cause: Throwable) -> Unit> {
```

Comment:
Is it only when a call has failed? Or it also triggers when the call is unhandled?

Reply:
Maybe we should think about naming, then?
Like `ExceptionHeppened` or `Exception` (it will be `on(Exception) { ... }`) or `CallFailedWithException`, etc.